rust/crd/src/lib.rs

```diff
@@ -2,18 +2,63 @@ pub mod druidconnection;
 pub mod supersetdb;
 
 use std::collections::BTreeMap;
+use std::num::ParseIntError;
 
 use serde::{Deserialize, Serialize};
+use snafu::Snafu;
 use stackable_operator::kube::runtime::reflector::ObjectRef;
 use stackable_operator::kube::CustomResource;
+use stackable_operator::product_config::flask_app_config_writer::{
+    FlaskAppConfigOptions, PythonType,
+};
 use stackable_operator::product_config_utils::{ConfigError, Configuration};
 use stackable_operator::role_utils::{Role, RoleGroupRef};
 use stackable_operator::schemars::{self, JsonSchema};
-use strum::{Display, EnumIter};
+use strum::{Display, EnumIter, EnumString, IntoEnumIterator};
 
 pub const APP_NAME: &str = "superset";
 pub const MANAGED_BY: &str = "superset-operator";
 
+pub const SUPERSET_CONFIG_FILENAME: &str = "superset_config.py";
```

Comment:
Just a hint: Be prepared for lot's of merge conflicts with https://github.com/stackabletech/superset-operator/pull/145

Reply:
I am glad that I am not responsible for #145. ;)

Reply:
Depends on who gets it's PR merged first höhö